### PR TITLE
Update browser monitoring doc on cookies usage post-v1216 NEWRELIC-4313

### PR DESCRIPTION
**Pull request by cwli24**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
**Revise existing documentation for browser monitoring due to new agent.**

* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
**Engineering change - agent no longer using cookies to track session.**
* If your issue relates to an existing GitHub issue, please link to it.
**N/a**

### CODE REVIEW

File: src/content/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking.mdx [hunk 1]
```diff
@@ -13,25 +13,25 @@ redirects:
...
+2. As the user's page is loaded, our browser monitoring script sets a simple session identifier in the browser.
+3. The script then reads the ID on subsequent page loads, reloads and restores, within the same tab.
+4. The same session will continue until the user's tab/window is closed.
 
 Session tracking will not work properly in these situations:
 
-* If users have cookies disabled in their browser
-* If the browser is configured to **not** allow information associated with the New Relic cookie to be sent with the request from the monitored site
+* If users have DOM Storage disabled in their browser.
+* If the browser or page is configured to **not** allow loading and execution of third-party origin scripts, which naturally means the agent as a whole will not function, through security policies or other means.
```
Comment (rhetoric101) [inline src/content/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking.mdx:30]: @cwli24 could we rearrange this sentence like this?

If the browser or page is configured to **not** allow loading and execution of third-party origin scripts through security policies or other means (which naturally means the agent as a whole will not function).

**Review by rhetoric101 — DISMISSED** [review]
@cwli24 These changes look good! I just made one suggestion you might consider.

File: src/content/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking.mdx (hunk 1 above)
Comment (cwli24) [reply src/content/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking.mdx]: Updated!

Comment (rhetoric101) [reply src/content/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking.mdx]: Thanks!

**Comment by rhetoric101** [comment]
@cwli24 GitHub is still asking you to sign the CLA. Even if you've signed it in the past, could you sign it again since we've been having trouble with our CLA records?

**Review by rhetoric101 — APPROVED** [review]
@cwli24 thanks for making these updates!

**Pull request merged**